src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs

```diff
@@ -13,6 +14,35 @@ namespace Microsoft.AspNetCore.Server.IntegrationTesting
     /// </summary>
     public class DeploymentParameters
     {
+        public DeploymentParameters()
+        {
+            EnvironmentVariables["ASPNETCORE_DETAILEDERRORS"] = "true";
+
+            var configAttribute = Assembly.GetCallingAssembly().GetCustomAttribute<AssemblyConfigurationAttribute>();
+            if (configAttribute != null && !string.IsNullOrEmpty(configAttribute.Configuration))
+            {
+                Configuration = configAttribute.Configuration;
+            }
+        }
+
+        public DeploymentParameters(TestVariant variant)
+        {
+            EnvironmentVariables["ASPNETCORE_DETAILEDERRORS"] = "true";
```

Comment:
Call other constructor and then set variant information.

Reply:
I'm a little worried about that messing up GetCallingAssembly(). I plan to get rid of most of these overloads. I primarily left them to ease the transition.